build.gradle

```diff
@@ -65,6 +65,13 @@ configurations {
     testCompile.exclude group: 'log4j', module: 'log4j'
 }
 
+test {
+    testLogging {
+        events = ["passed", "failed", "skipped"]
+        showStandardStreams = false
```

Comment:
We can change this flag to true to see the `stdout` results on console but the results are verbose and hence turned off by default.